src/math-land/invert.py

```diff
@@ -0,0 +1,103 @@
+from typing import Self
+
+
+class PrimaryColor:
+    """Base class for each primary color in the RGB space
+
+    >>> c = PrimaryColor(100)
+    >>> print(c)
+    100
+
+    >>> c.another_attr = 200
+    Traceback (most recent call last):
+        ...
+    AttributeError: 'PrimaryColor' object has no attribute 'another_attr'
+    """
+
+    __slots__ = ("value",)
+
+    def __init__(self, value: int) -> None:
+        """
+        Inititalize and check if the PrimaryColor has a valid value
+
+        >>> PrimaryColor(888)
+        Traceback (most recent call last):
+            ...
+        ValueError: Color value is not between 0 and 255
+        """
+
+        if not 0 <= value <= 255:
+            raise ValueError("Color value is not between 0 and 255")
+
+        self.value = value
+
+    def __invert__(self) -> Self:
+        """
+        Returns the inverse of the color as the complement to 255
+
+        >>> print(~PrimaryColor(100))
+        155
+        """
+        return PrimaryColor(255 - self.value)
+
+    def __str__(self) -> str:
+        return str(self.value)
+
+
+class RGBColor:
+    """
+    A color in RGB space.
```

Comment:
```suggestion
    """A color in RGB space
```
Notice I also removed the period at the end, to be consistent with the others.